### ISSUE TITLE
Allow hierarchical roles on the Convention Contacts page

### ISSUE THREAD

**Issue by jhartwin** [neinteractiveliterature/intercode#1952]
I'd like to be able to list some hierarchical roles on the Convention Contacts page; e.g.:
Iron GM Coordinator
> Iron GM At-con Logistics Specialist
Ops Coordinator
> Room Divider Wrangler
etc.

**Comment by nbudin** [comment]
Seems reasonable; staff positions can have a parent staff position.  I think maybe we should only let this go one level deep for now; we can always revisit that.

**Comment by jhartwin** [comment]
Makes sense. As the con grows we may need more levels, but certainly for at least the next few years one will likely be sufficient.
